pages/test_analytics/ruby_collectors.md.erb

```diff
@@ -99,6 +99,20 @@ Once you're done, in your Test Analytics dashboard, you'll see analytics of test
 
 If you don't see branch names, build numbers, or commit hashes in the Test Analytics UI, then see [CI Environments](/docs/test-analytics/ci-environments) to learn more about exporting your environment to the minitest collector.
 
+## Adding annotation spans
+
+This gem allows adding custom annotations to the span data sent to Buildkite using the [.annotate](https://github.com/buildkite/test-collector-ruby/blob/d9fe11341e4aa470e766febee38124b644572360/lib/buildkite/test_collector.rb#L64) method. For example:
+
+```ruby
+Buildkite::TestCollector.annotate("Rendered OTP Screen")
+```
+
+This would appear like so:
+
+<%= image "annotation-span.png", width: 512, height: 359, alt: "Screenshot of the span timeline including the 'Rendered OTP Screen' annotation." %>
+
+This is particularly useful for tests that generate a lot of span data such as system/feature tests.
+
```

Comment:
Question: do we need to tell customers where to find span timeline in the UI?

I'm thinking, for new customers who have just set up the collector and have not collected any data yet, it may not be immediately clear where they can look up this information.

Reply:
FYI, if we want to elaborate to customers where to find span timeline in the app — a reminder to italicize all references to UI elements.

[See Docs styleguide](https://github.com/buildkite/docs/blob/main/styleguide/STYLE.md#referring-to-ui-elements)

Thanks for bearing with my nitpicking 🙏  please shout out if you'd like some help.

Reply:
I had trouble thinking of some good wording for this. How about:

> The span timeline is displayed on the execution page and includes all annotations.

Any better suggestions?

Reply:
> Thanks for bearing with my nitpicking pray please shout out if you'd like some help.

No problem. Clear correct documentation is important 👍️ I'll definitely do my best but I might need some help.

Reply:
Thanks @swebb :) Re https://github.com/buildkite/docs/pull/1738#discussion_r985343052

What do you think about:

> You can find all _annotations_ under _Span Timeline_ at the bottom of every _test execution page_.